src/components/NcCounterBubble/NcCounterBubble.vue

```diff
@@ -87,8 +87,9 @@ export default {
 	line-height: 1em;
 	padding: 4px 6px;
 	border-radius: var(--border-radius-pill);
-	background-color: var(--color-background-darker);
+	background-color: var(--color-primary-light);
 	font-weight: bold;
+	color: var(--color-primary);
```

Comment:
```suggestion
	color: var(--color-primary-element);
```

Reply:
> I'd suggest that we use the primary-element colors so that if the primary color is a light grey for example, we fallback to a value that still has enough contrast.

yess, on it